cdap-data-fabric/src/test/java/co/cask/cdap/data2/dataset2/lib/partitioned/PartitionedFileSetTest.java

```diff
@@ -211,6 +211,21 @@ public void testPermissions() throws Exception {
     Assert.assertEquals(tablePermissions, TableProperties.getTablePermissions(props));
   }
 
+  @Test
+  public void testDefaultBasePath() throws Exception {
+    DatasetId id = DatasetFrameworkTestUtil.NAMESPACE_ID.dataset("testDefaultPath");
+    dsFrameworkUtil.createInstance("partitionedFileSet", id, PartitionedFileSetProperties.builder()
+      .setPartitioning(PARTITIONING_1)
+      .build());
+    PartitionedFileSet pfs = dsFrameworkUtil.getInstance(id);
+    Location baseLocation = pfs.getEmbeddedFileSet().getBaseLocation();
+    Assert.assertEquals(baseLocation.getName(), id.getDataset());
+    Assert.assertTrue(baseLocation.exists());
+    Assert.assertTrue(baseLocation.isDirectory());
+    dsFrameworkUtil.deleteInstance(id);
+    Assert.assertFalse(baseLocation.exists());
```

Comment:
I think this would be true with the old behavior, too. If the base location is <name>/files, then that will be deleted...

To really test that it behaves correctly, I think you can create both a file set and pfs, get both base locations and assert that their parent is the same?